scripts/src/main/resources/scripts/command/graalvm

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: optional setup
+function doSetup() {
+  GRAALVM_HOME="${DEVON_IDE_HOME}/software/extra/graalvm"
+  doDevonCommand java setup silent
+  if [ ! -d "${GRAALVM_HOME}" ]
+  then
+    local version="${GRAAL_VERSION:-22.0.0.2}"
+    local javaVersion="${EXTRA_JAVA_VERSION}"
```

Comment:
Great that you look into other commandlets and apply existing solutions...
So does GraalVM really require a JDK (with a version >= 11)? I was not fully aware of this. Do you have a link for this fact? 
I only found this what seems to cause other issues we might have to cover:
https://www.graalvm.org/22.0/reference-manual/native-image/#prerequisites
If it requires JDK >= 11 it is prefect that you are checking this here.
However, I would suggest that we use `EXTRA_JAVA_VERSION` only for IDEs and for all build related stuff we should ignore it and just honor `JAVA_VERSION` and the default Java (which will be on the PATH, in JAVA_HOME and physically in `software/java`).

Reply:
I understood that because for the download of GraalVM there are two versions: for 11 and 17, so I thought that meant that you had to have version 11 or 17. I chose 11 so as not to require 17. Maybe I'am wrong.

Base JDKs (GraalVM Community) = OpenJDK 11.0.13 and 17.0.1

Source:
https://www.graalvm.org/downloads/
https://github.com/graalvm/graalvm-ce-builds/releases